src/main/java/com/minecolonies/coremod/util/RecipeHandler.java

```diff
@@ -28,6 +28,10 @@ private RecipeHandler()
      */
     public static void init(final boolean enableInDevelopmentFeatures, final boolean supplyChests)
     {
+        GameRegistry.addRecipe(new ItemStack(ModBlocks.blockConstructionTape, 16), "SWS", "S S", "S S",
+          'S', Items.STICK, 'W', new ItemStack(Blocks.WOOL, 1,4));
```

Comment:
we should have here the color constant to spell what this number means as a color